spec/numberformat.html

```diff
@@ -103,6 +103,22 @@
       </p>
     </emu-clause>
 
+    <emu-clause id="sec-formatnumberstring" aoid="FormatNumberString">
+      <h1>FormatNumberString ( _numberFormat_, _x_ )</h1>
```

Comment:
I don't particular like this name, because it tells me nothing. Maybe `FormatNumberToString`